modtools/set-belief.lua

```diff
@@ -0,0 +1,460 @@
+-- Changes the beliefs (values) of units.
```

Comment:
Just realized this sounds similar to `assign-belief` from #105 - haven't really looked into how similar they are, but we might need to consolidate them.
None of your other scripts sound as similar, but there could be some overlap too.

Reply:
Hehe yeah, that's the one I was alluding to :P

Reply:
Since neither are merged yet, I'm thinking I'll take the ones from #105 and make sure any extra features from these make it in too - sound ok?

Reply:
If you want, but you'd be having to add quite a lot of features. It's awkward that they both have overlap and dependencies on themselves. It might be dumb but you could always include both, it's not like the regular scripts aren't without redundancies (or at least some will be with the inclusion of either of these) :P

Reply:
The gm-unit features are neat, and I don't feel like reimplementing those using the other scripts, so I'll merge this as-is and we can refactor it later.